src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const listGallery = document.querySelector('.gallery__list');
+
+listGallery.addEventListener('click', (currentElement) => {
+  const link = currentElement.target.closest('a');
+
+  if (link !== null) {
+    largeImage.src = link.href;
+  }
+
+  currentElement.preventDefault();
```

Comment:
The `preventDefault()` method should be called on the event object to prevent the default action of the anchor tag. Ensure that `currentElement` is indeed the event object, or rename it to `event` for clarity.